image-filters/image-filters.markdown

```diff
@@ -573,56 +570,56 @@ of colors in a program is very important to understanding how our filters work.
+opacity, where 0 is transparent and 100% is opaque.
 
 #### HSB or HSV colors
-This color space is not quite as well known as RGB, the first number represents
-the hue, the second the saturation (how intense the color is), and the third
+This color space is not quite as well known as RGB. The first number represents
+the hue, the second number the saturation (how intense the color is), and the third
 number the brightness. The image at the top was created by manipulating in this
-color space. The HSB color space can be represented by drawing a cone. The hue
+color space. The HSB color space can be represented by a cone: The hue
 is the position around the cone, saturation the distance from the centre, and
 brightness the height (0 brightness is black).
 
-### Extracting The Dominant Hue From an Image
-Now we’re comfortable with pixel manipulation, let’s do something that we could
-only do digitally. Digitally, we can manipulate it in a way that isn’t so
+### Extracting the Dominant Hue from an Image
+Now that we’re comfortable with pixel manipulation, let’s do something that we could
+only do digitally. Digitally, we can manipulate the image in a way that isn’t so
 uniform.
 
-When I look through my stream of pictures, say on Flickr, I can see themes
-emerging. The nighttime series from the boat I took at sunset around Hong Kong
+When I look through my stream of pictures I can see themes
+emerging. The nighttime series I took at sunset from a boat on Hong Kong
 harbour, the grey of North Korea, the lush greens of Bali, the icy whites and
 pale blues of an Icelandic winter. Can we take a picture and pull out that main
 color that dominates the scene?
 
-It makes sense to use the HSB color space for this - we are interested in the
-hue, not the saturation or brightness, when figuring out what the main color
+It makes sense to use the HSB color space for this — we are interested in the
+hue when figuring out what the main color
 is. It’s possible to do this using RGB values, but more difficult (we would
 have to compare all three values) and it would be more sensitive to darkness.
-We can change to this colorspace using
+We can change to the HSB color space using
 [colorMode](http://processing.org/reference/colorMode_.html). 
 
 Having settled on this color space, it’s simpler than it would have been using
 RGB. We need to find the hue of each pixel, and figure out which is most
-"popular". We probably don’t want to be exact, we want to group very similar
+"popular". We probably don’t want to be exact — we want to group very similar
 hues together, and we can handle this using two strategies.
 
 Firstly we will round the decimals that come back to whole numbers, as this
 makes it simple to determine which "bucket" we put each pixel in. Secondly we
-can change the range of the hues. If we think back up to the diagram above, we
+can change the range of the hues. If we think back to the cone representation above, we
```

Comment:
I wasn't sure what diagram this was, so I changed the wording --- please suggest other wording if I have missed the point!